src/main/java/frc/imaging/Limelight.java

```diff
@@ -18,7 +19,7 @@ public static Limelight getInstance() {
     }
 
     public enum LimelightTargetType {
-        POWER_PORT, LOADING_BAY
+        UPPER_HUB, LOADING_BAY, POWER_PORT
```

Comment:
Are there more vision targets? I thought there were only retro-reflective tape on the UPPER_HUB

Reply:
Only the UPPER_HUB, but I kept the other two for practice and troubleshooting!